KBaseSearchEngine.spec

```diff
@@ -70,9 +70,9 @@ module KBaseSearchEngine {
     /*
       Optional rules of access constraints.
         - with_private - include data found in workspaces not marked 
-            as public, default value is true,
+                    as public, default value is true for authenticated users and false for unauthenticated users.
         - with_public - include data found in public workspaces,
-            default value is false,
+                    default value is false for authenticated users and true for unauthenticated users.
```

Comment:
Based on the code this is not actually settable by an unauthed user. It'll always be forced to true.